pages/careers/depCreationPage.vue

```diff
@@ -0,0 +1,138 @@
+<template>
+  <section class="relative md:py-24 py-16">
+    <div class="container">
+      <div class="grid lg:grid-cols-12 grid-cols-1" id="reserve-form">
+        <div class="lg:col-start-2 lg:col-span-10">
+          <div
+            class="rounded-md shadow dark:shadow-gray-800 bg-white dark:bg-slate-900 p-6"
+          >
+            <form @submit.prevent="submitForm">
+              <div class="grid lg:grid-cols-12 gap-6">
+                <div class="lg:col-span-6">
+                  <div class="ltr:text-left rtl:text-right">
+                    <label for="name" class="font-semibold">Name:</label>
+                    <input
```

Comment:
We have to use existing component instead of input

Reply:
This is what is available in a reuseable components file. you can check.